src/app/ubs/ubs-admin/components/ubs-admin-order/ubs-admin-order.component.ts

```diff
@@ -547,7 +548,11 @@ export class UbsAdminOrderComponent implements OnInit, OnDestroy, AfterContentCh
   }
 
   statusCanceledOrDone(): void {
-    if (this.currentOrderStatus === 'CANCELED' || this.currentOrderStatus === 'DONE') {
+    const exportDetails = this.orderForm.get('exportDetailsDto').value;
+    const allFieldsHaveValue = Object.keys(exportDetails).every((key) => exportDetails[key] != null && exportDetails[key] !== '');
```

Comment:
Can we just use exportDetails[key] for the conditions here?

Reply:
yep, It will be simpler, thanks